NFproject/articles/templates/article.html

```diff
@@ -1,10 +1,13 @@
 {% extends 'base.html' %}
 {% block content %}
 <div class="container">
-  <h1> {{article.title}}</h1>
+  <h1> {{article.Title}}</h1>
   <p>
 {{article.content}}
 </p>
 
+<h3>{{article.Author}}</h3>
+<h4>{{article.Date_created}}</h4>
+<img src={{article.pic}}>
```

Comment:
Avoid the urge to abbreviate field names.

`article.picture` or `article.image`